### DISCUSSION TITLE
fatal error: error in backend: Cannot select: intrinsic %llvm.nvvm.implicit.offset

### DISCUSSION THREAD

**Discussion by zjin-lcf** [intel/llvm#4489] [General]
Could you explain the cause of the build error "Cannot select: intrinsic %llvm.nvvm.implicit.offset" for a SYCL program ?

Thanks

**Comment by Michoumichmich** [comment]
I believe that it happens when you don't compile with `nvptx64-nvidia-cuda`, but with `nvptx64` or `nvptx64-unknown-unknown`. All the instructions available are not the same

**Reply by zjin-lcf** [reply]
Thanks!
